tests/foreman/cli/test_import.py

```diff
@@ -399,6 +423,9 @@ def tearDown(self):
             u'rm -rf "${HOME}"/.transition_data "${HOME}"/puppet_work_dir'
         )
 
+    def runTest(self):
```

Comment:
Could you please explain the purpose of this func? I can't see any calls for this in your tests. I suppose you override some unittest func then? If so, why don't you call `super`?